src/views/address/address.html

```diff
@@ -60,15 +60,25 @@ <h1 id="header">{{translate("pages.address-form.title")}}</h1>
       value: addressLocality
     }) }}
 
-    <h4>{{translate("fields.addressYearFrom.title")}}</h4>
+    {% if values.addPreviousAddresses %}
```

Comment:
I have another PR out which will change this - https://github.com/alphagov/di-ipv-cri-address-front/pull/123